tests/explainers/test_tree.py

```diff
@@ -610,37 +530,29 @@ def test_lightgbm_binary():
 # TODO: Test tree_limit argument
 
 def test_sklearn_interaction():
-    import sklearn
-    from sklearn.model_selection import train_test_split
-    from sklearn.ensemble import RandomForestClassifier
-    import numpy as np
-    import shap
-
     # train a simple sklean RF model on the iris dataset
     X, _ = shap.datasets.iris()
-    X_train,_,Y_train,_ = train_test_split(*shap.datasets.iris(), test_size=0.2, random_state=0)
-    rforest = RandomForestClassifier(n_estimators=100, max_depth=None, min_samples_split=2, random_state=0)
+    X_train, _, Y_train, _ = sklearn.model_selection.train_test_split(*shap.datasets.iris(),
+                                                                      test_size=0.2, random_state=0)
+    rforest = sklearn.ensemble.RandomForestClassifier(n_estimators=100, max_depth=None,
+                                                      min_samples_split=2,
+                                                      random_state=0)
     model = rforest.fit(X_train, Y_train)
 
     # verify symmetry of the interaction values (this typically breaks if anything is wrong)
     interaction_vals = shap.TreeExplainer(model).shap_interaction_values(X)
-    for i in range(len(interaction_vals)):
```

Comment:
Is this change necessary? Feels like range(len) is already fine here. Same for below

Reply:
This one is a pylint failure. I agree that there is not much difference, but IMO it's better to conform.

Reply:
pylint is only really a list of guidelines, and doesn't necessarily have to be followed religiously. Remember that every change takes time both to make, and to have someone else review, so if there is no difference either way my opinion would always be to leave it how it is.